### DISCUSSION TITLE
Build is Crashing on auto CI/CD github

### DISCUSSION THREAD

**Discussion by shaniKazi** [webpack/webpack#17571] [Q&A]
# Bug report

**What is the current behavior?**
I am facing this issue on ci github build and deploy  for the react 
ERROR in vendor.866ce385a84cdb53c394.chunk.js from Terser
Error: Call retries were exceeded

**If the current behavior is a bug, please provide the steps to reproduce.**
I am using webpack 
  "webpack": "4.46.0",
     "terser-webpack-plugin": "^4.2.3",

**What is the expected behavior?**

**Other relevant information:**
webpack version:  4.46.0
Node.js version: 16.15.0
Operating System: ubuntu-latest
Additional tools:

**Comment by snitin315** [comment]
Thank you for creating this issue. However, issues need to follow one of our templates so that we can clearly understand your particular circumstances.

Please help us help you by recreating the issue using one of our templates.
